api/app/src/sequelize/migrations/2023050403101805-update-patient-data.ts

```diff
@@ -0,0 +1,70 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { PatientModel } from "../../models/medical/patient";
+
+// Use 'Promise.all' when changes are independent of each other
+// Docs: https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    const patients = (await queryInterface.select(null, PatientModel.NAME, {
+      transaction,
+    })) as any[]; //eslint-disable-line @typescript-eslint/no-explicit-any
+    const newPatients: any[] = []; //eslint-disable-line @typescript-eslint/no-explicit-any
+    for (const patient of patients) {
+      let newPatientData = { ...patient.data };
```

Comment:
NABD as it seems we're not relying on `patient` not to be updated in memory, but still worth aligning on this here:
- the code above suggests we want to copy the contents of `patient` - otherwise we could just refer one "variable" to another;
- the decomposition operator does a shallow copy, which is fine for some situations - but here we have at least 2 additional levels if I'm not wrong;
- Lodash has a function for these situations, `cloneDeep()`

Reply:
updated to avoid copy